README.md

```diff
@@ -647,8 +649,8 @@ controller.cancel();
 
 ## Uploading Local Files
 
-The library supports uploading local files and directories. For directories, the tool
-will include all immediate child files in the directory. It will not process sub-directories.
+The library supports uploading local files and folders. For folders, the tool
+will include all immediate child files in the folder. It will not process sub-folders unless the "deep upload" option is specified.
```

Comment:
```suggestion
will include all immediate children files in the folder. It will not process sub-folders unless the "deep upload" option is specified.
```